packages/app/src/locales/en.json

```diff
@@ -591,7 +591,15 @@
         "table": {
             "tokenName": "Token Name",
             "price": "Price",
-            "tokenAddress": "Token Address"
+            "tokenAddress": "Token Address",
+            "bridged": {
+                "title": "Default bridge",
```

Comment:
What if for some tokens it's not the default bridge? Maybe better to say just `Bridged`?